### ISSUE TITLE
Cannot start debugger for .NetCore project

### ISSUE THREAD

**Issue by tbvl** [GoogleCloudPlatform/cloud-code-vscode#225]
**Environment**:
Extension version: 1.2.1
VSCode version: 1.44.0
OS: darwin
ms-dotnettools.csharp: 1.21.16

**Description**:
When starting the debugger for .NET core project you get "_Extensions ms-vscode.csharp needed for debugging. Please install or enable them."_ but it is not possible to find extension in market place with that identifier.
The extension has changed marketplace ID from ms-vscode.csharp to ms-dotnettools.csharp in release 1.21.13 on March 5, 2020, see https://marketplace.visualstudio.com/items/ms-dotnettools.csharp/changelog.

So please change dependency in cloud code extension as well, or make it configurable for backwards compatibility, thanks!

**Repro step**:
Follow the quick start guide at: https://cloud.google.com/code/docs/vscode/quickstart, and select .NET core "Hello World Application" example. 
At step: https://cloud.google.com/code/docs/vscode/quickstart#debugging_your_app you get the following pop-up:
![image](https://user-images.githubusercontent.com/1332978/79069887-183f4f00-7cd2-11ea-8a23-6770d346e14d.png)

This is the configuration in launch.json (standard one):
``` json
(...)
{
        "name": "Attach to Kubernetes pod",
        "type": "cloudcode.kubernetes",
        "request": "attach",
        "podSelector": {
          "app": "dotnet-hello-world",
        },
        "localRoot": "${workspaceFolder}/src",
        "language": "NETCore",
        "remoteRoot": "/src",
        "debugPort": 9229
      }
(...)
```

**Comment by quoctruong** [comment]
@tbvl Thank you for reporting the issue. The fix for this will be available in the next release.

**Comment by kschaab** [comment]
Just released v1.3.0 which should address this issue.
